improver_tests/acceptance/SHA256SUMS

```diff
@@ -363,7 +367,7 @@ f7f403777278b966cd31e3db8592e5f30160076222e57891c67b9cfd3f5ee8f0  ./resolve-wind
 77d35d36e52a65aa782e969e791f79c5558be8b0cf838fdc00bef3b4e75e6723  ./sleet_probability/basic/half_prob_snow_falling_level.nc
 e592f74bec62c53fe906d6f1c1d87f4acec5f2660457c0db98ffc00c64066663  ./sleet_probability/basic/kgo.nc
 efff8720d452591b6531a279bad6c0168db2336fa497c1f1087ecfcb9321af92  ./sleet_probability/basic/tenth_prob_snow_falling_level.nc
-b1a29b84e460af5712ea74b7d8709c14e00f7b5f6f710b78e24567a2602598e1  ./snow-fraction/basic/kgo.nc
+29f807dbfdc6ffea2125cf57b31fbc417c9f80dfa38dca37349878256b7508db  ./snow-fraction/basic/kgo.nc
```

Comment:
Maybe I'm being dense - why did this change?

Reply:
Because of the aforementioned masked data. I discovered that unmasked rain and snow data were being translated by the NetCDF reader into cubes with masked Numpy arrays (and the whole mask set to False). Numpy handles divide-by-zero differently if the inputs are normal arrays (np.nan) or masked arrays (masked=True). This in turn impacts the interpolation scheme (masked data are not filled, np.nan are filled). So the KGO that existed had masked data points everywhere where both rain and snow were zero. This couldn't be accepted by the phase-mask plugin so I had to investigate and fix. 
Also, I've added the model_id_attr attribute.

Reply:
OK, thanks for the explanation.